src/collects/seashell/backend/asan-error-parse.rkt

```diff
@@ -248,24 +271,23 @@
 (define frame-parse : SectionParser
   (match-and-process
    #px"^\\{\"frame\": "
```

Comment:
This line is the reason why there was no stack frame. Originally, ASAN output was configured to print the stack frame in a JSON string that looks like 

```
{"frame": 0, "module": "/home/yc2lee/.seashell/runtime-files/A7-test-seq.c-g37-binary", "offset": "0x41ca96", "function": "atomic_compare_exchange_strong<__sanitizer::atomic_uint8_t>", "function_offset": "0x0", "file": "/home/yc2lee/seashell/lib/llvm/projects/compiler-rt/lib/asan/../sanitizer_common/sanitizer_atomic_clang.h", "line": 81, "column": 0}
{"frame": 1, "module": "/home/yc2lee/.seashell/runtime-files/A7-test-seq.c-g37-binary", "offset": "0x41ca96", "function": "__asan::Allocator::AtomicallySetQuarantineFlag(__asan::AsanChunk*, void*, __sanitizer::BufferedStackTrace*)", "function_offset": "0x0", "file": "/home/yc2lee/seashell/lib/llvm/projects/compiler-rt/lib/asan/asan_allocator.cc", "line": 466, "column": 0}
{"frame": 2, "module": "/home/yc2lee/.seashell/runtime-files/A7-test-seq.c-g37-binary", "offset": "0x41ca96", "function": "__asan::Allocator::Deallocate(void*, unsigned long, __sanitizer::BufferedStackTrace*, __asan::AllocType)", "function_offset": "0x0", "file": "/home/yc2lee/seashell/lib/llvm/projects/compiler-rt/lib/asan/asan_allocator.cc", "line": 526, "column": 0}
```

Then in

https://github.com/cs136/seashell/pull/563/files#diff-356bc6d909635a7ef49ced55bbff242dL326

I switched to using the default ASAN stack frame output format which looks like this instead:

```
    #0 0x4cdcad in f /home/yc2lee/.seashell/projects/A7/q1/test-seq.c:2:23
    #1 0x4cdcb8 in g /home/yc2lee/.seashell/projects/A7/q1/test-seq.c:3:12
    #2 0x4cdd41 in main /home/yc2lee/.seashell/projects/A7/q1/test-seq.c:5:3
    #3 0x7f9c292c8f44 in __libc_start_main /build/eglibc-oGUzwX/eglibc-2.19/csu/libc-start.c:287
    #4 0x417b66 in _start (/home/yc2lee/.seashell/runtime-files/A7-test-seq.c-g35-binary+0x417b66)
```

The `try-parse-stack-line` function below is matching the default format, while `frame-parse` is still matching the custom {frame...} JSON string format.